src/moodleapi/Moodleapi.ts

```diff
@@ -0,0 +1,39 @@
+interface Response {
```

Comment:
This should be something more specific since we may have other methods later with different response shapes. Maybe the interface name can be `GetUserResponse` :man_shrugging: ?